docs/source/conf.py

```diff
@@ -463,3 +464,40 @@ def setup(app):
     # This will also rebuild appropriately when the value changes.
     app.add_config_value('cuda_enabled', cuda_enabled, 'env')
     app.add_config_value('flight_enabled', flight_enabled, 'env')
+    app.add_directive('computefuncs', ComputeFunctionsTableDirective)
+
+
+class ComputeFunctionsTableDirective(Directive):
+    has_content = True
+    option_spec = {
+        "kind": directives.unchanged
+    }
+
+    def run(self):
+        from docutils.statemachine import ViewList
+        from docutils import nodes
+        import pyarrow._compute
```

Comment:
Is there a reason for importing this instead of the public `pyarrow.compute`?

Reply:
Just because I was using `get_function_registry`, I'll switch to use the compute module itself.